File: python/strings/Sorting_the_Sentence_1859.py
class Solution:
    def sortSentence(self, s: str) -> str:
        words=s.split()
        res=[None]*len(words)
    
        for word in words:
            index=int(word[-1])
            correct_pos=index-1
            res[correct_pos]=word[:-1]
        return " ".join(res)

File: java/strings/Sorting_the_Sentence_1859.java
package java.strings;

public class Sorting_the_Sentence_1859 {
     public String sortSentence(String s) {
        String[] words=s.split(" ");
        String[] res=new String[words.length]; 
        for (String word : words){
            int index=word.charAt(word.length()-1)-'0';
            int correct_index=index-1;
            String actual_word=word.substring(0,word.length()-1);
            res[correct_index]=actual_word;
        }
        return String.join(" ",res);

    }
}
